test/Table_Tests/data/.gitignore

```diff
@@ -0,0 +1 @@
+*.bak
```

Comment:
The tests may fail and so they will still sometimes leave out garbage because of that. We can add some 'global' after test cleanup, but that may still fail if the tests fail in some "fatal" way. So I'd rather not assume a clean environment. But indeed maybe it makes sense to do a bit more cleanup after the tests.
@jdunkerley what do you think?

One reason which makes me hesitant is that actually by _not_ cleaning the files, I can sometimes do 'post-mortem debugging' by looking at the content of files generated during the tests. And this has helped me a few times already.

Reply:
We should clean up as part of the test but it does fail from time to time.

Adding to `.gitignore` seems sensible as well though.

Reply:
Fair enough